packages/cluster-ui/src/statementsPage/statementsPage.tsx

```diff
@@ -126,12 +127,14 @@ export class StatementsPage extends React.Component<
     const searchParams = new URLSearchParams(history.location.search);
     const sortKey = searchParams.get("sortKey") || undefined;
     const ascending = searchParams.get("ascending") || undefined;
+    const columnTitle = searchParams.get("columnTitle") || undefined;
     const searchQuery = searchParams.get("q") || undefined;
 
     return {
       sortSetting: {
         sortKey,
-        ascending: Boolean(ascending),
+        ascending: ascending === "true",
```

Comment:
`Boolean(ascending)`is not the correct way to change from string to boolean, what `Boolean()` does is check if there is any value for that parameter, and that was in fact the bug, because it always return true